packages/test/test-end-to-end-tests/src/test/benchmark/container.time.spec.ts

```diff
@@ -43,18 +43,16 @@ describeNoCompat("Container - runtime benchmarks", (getTestObjectProvider) => {
 
 	before(function () {
 		provider = getTestObjectProvider();
-
-		// TODO: Convert these to mocked unit test. These are all API tests and doesn't
-		// need the service.  For new disable the tests other than local driver
-		if (provider.driver.type !== "local") {
-			this.skip();
-		}
```

Comment:
Since this part is gone we can probably simplify the file a bit, move `provider = getTestObjectProvider();` to line 24 and make it a `const`. In both files.
I'm not sure why that part had to be inside this `before()`...

Reply:
I've removed the before and will probably do some refactoring in the near future - to see if we can merge the files. 
I will chat with you about the idea later on :)